snap/snapcraft.yaml

```diff
@@ -152,6 +152,17 @@ parts:
       - usr/share/doc/*/copyright
       - usr/share/egl/egl_external_platform.d
 
+  gbm:
+    # GBM EGL external platform
+    plugin: nil
+    stage-packages:
+      - to amd64:
+        - libnvidia-egl-gbm1
+      - to arm64:
+        - libnvidia-egl-gbm1
```

Comment:
It's also there on `ppc64el`

Reply:
It is, but `$libdir/gbm/nvidia-drm_gbm.so` is provided by the nvidia driver which only builds for amd64 and arm64.